subsys/partition_manager/Kconfig

```diff
@@ -34,6 +34,17 @@ config SRAM_BASE_ADDRESS
 
 menu "Zephyr subsystem configurations"
 
+DT_CHOSEN_Z_IPC_SHM := zephyr,ipc_shm
+
+config RPMSG_NRF53_SRAM_SIZE
+	hex "RPMsg shared memory size"
+	default "$(dt_chosen_reg_size_hex,$(DT_CHOSEN_Z_IPC_SHM))"
+	depends on BT_RPMSG_NRF53 && SOC_NRF5340_CPUAPP
+	depends on PARTITION_MANAGER_ENABLED
```

Comment:
Shouldn't it be defined also if Partition Manager is disabled? This one relies on DTS.

Reply:
Zephyr has renamed the kconfig option used for the buffer size, so this is actually no longer used by anyone except NCS in partition manager context. There is no point in showing this value to the user unless partition manager is enabled as it is not used then.